pulumi/src/index.ts

```diff
@@ -27,15 +27,19 @@ export interface BaseConfig {
   rootDomainName?: string
 }
 
+export interface StsDefinition {
+  replicas: number
+  services: Array<ServiceConfig>,
+  backupSchedule?: string
+  backupCount?: number
```

Comment:
```suggestion
  backup?: {
    schedule: string
    backupCount: number
  }
```

Reply:
Will also want to add default/validation to `pulumi/src/config.ts` for new `backup` config